packages/documentation/src/pages/architecture/unit-tests.mdx

```diff
@@ -0,0 +1,120 @@
+## How do unit tests work?
+
+#### Where they live
+- Tests can live anywhere in the `src` directory, but in general are located in a directory close to the code or application that they are written to test.
```

Comment:
We should remove the compound sentence and put the most pertinent information at the beginning. 

> Tests are generally co-located in the application directory with the files they test.